Userland/Libraries/LibWeb/Animations/Animatable.cpp

```diff
@@ -152,6 +152,11 @@ void Animatable::clear_transitions()
 {
     m_transition_attribute_indices.clear();
     m_transition_attributes.clear();
+
+    for (auto& property_id : m_associated_transitions.keys()) {
+        if (m_associated_transitions.contains(property_id))
+            m_associated_transitions.remove(property_id);
+    }
```

Comment:
Could you do: `m_associated_transitions.clear()` here instead?

Reply:
Changed to clear.

Reply:
Please squash your latest change into the previous commit rather than adding it as a separate commit. Thanks!

Reply:
Your updated code does the following:

```
for each (key, value in transitions_map)
  if transitions_map has a value for key
      remove every key,value pair from transitions_map
```

Compare your code to the code used to clear the other HashMap data member associated with transitions. It needs an update :)

Also, please make sure to fix-up your commits after review, instead of pushing more commits to the PR. If you don't know how to do that, refer to this video, or ask on discord. https://www.youtube.com/watch?v=ElRzTuYln0M

Reply:
Squashed the commits.